EIPS/eip-7702.md

```diff
@@ -43,6 +43,14 @@ rlp([chain_id, nonce, max_priority_fee_per_gas, max_fee_per_gas, gas_limit, dest
 authorization_list = [[chain_id, address, nonce, y_parity, r, s], ...]
 ```
 
+Transaction is considered invalid if authorization list items can't be decoded as:
```

Comment:
Invalid, as in should be rejected by txpool? (If yes I think this should be clarified - and I also think this should be the case :smile: )

Reply:
It's invalid as it can't be added to the block. And if invalid tx is found in the block, the block is considered invalid.

Valid/Invalid should be the correct term used to spec out this.

It is assumed that txpool rejects invalid transactions.

Reply:
Alright, yes sorry about this, I sometimes get confused about what terminology maps to what, since "invalid" could also be interpreted as the tx itself is valid but will be reverted immediately. (It would be great at some point to have a glossary EIP of the terminology we use to avoid confusion :) )

Reply:
> (It would be great at some point to have a glossary EIP of the terminology we use to avoid confusion :) )

true, this would be helpful